### PR TITLE
pm starting out

### LINKED ISSUE #103: Feature: PM Support
**What do you want to see?**
A way to view browse and respond to PM's.

**Additional context**
As an admin, you end up messaging a bunch of people in regards to various things
A page to view:
- [x] PMs received/sent PMs
- [x] Send new PM
- [x] #107
- [x] could have a notifications tooltip that just shows the unread ones.... 
- [ ]  OPTION: auto-mark read on view (default: false)

### PR DESCRIPTION
**Pull request by tgxn**
for #103
image

- threaded messaging support
- reply to thread support
- url parameter defines active pm

todoo
- [ ] could have a notifications tooltip that just shows the unread ones....
  - [x] fix the notifcation icon display, too
- [x] style fixes/adjustments padding/borders/etc
- [x] adjust message author and recipient display
- [x] display tooltip on unread messages to mark read
- [x] mark all in thread as read
- [ ] OPTION: auto-mark read on view `(default: false)`
- [x] order newest message at bottom and auto-scroll?
- [x] - hover should give menu with option to mark read
- [x] still not scrolling to bottom on inital load
- [x] Mention & Reply Notification Support
- [x] need to support message newlines and markdown

### CODE REVIEW

**Comment by tgxn** [comment]
image
threaded support

**Comment by Pdzly** [comment]
Good work!

File: src/components/Shared/UserChips.jsx [hunk 1]
```diff
@@ -58,22 +58,16 @@ export function SiteAdminChip({ person }) {
...
-      iconOnly={<SmartToyIcon fontSize="small" />}
-    />
-  );
+  return <SquareChip color={"warning"} tooltip="Bot Account" iconOnly={<SmartToyIcon fontSize="small" />} />;
 }
 export function DeletedUserChip({ person }) {
   if (!person.deleted) return null;
 
-  return <SquareChip color={"danger"} tooltip="User is deleted" iconOnly={<DeleteIcon fontSize="small" />} />;
+  return <SquareChip color={"danger"} tooltip="Beleted User" iconOnly={<DeleteIcon fontSize="small" />} />;
```
Comment (Pdzly) [inline src/components/Shared/UserChips.jsx:72]: Is it wanted to be "Beleted User?"

Comment (tgxn) [reply src/components/Shared/UserChips.jsx:72]: thanks :)

Comment (tgxn) [reply src/components/Shared/UserChips.jsx:72]: I'm also interested in maybe looking at I18N for this too - we'd have to pull ALL the strings out :/

**Pull request merged**